src/core/ui.js

```diff
@@ -70,6 +70,13 @@ respecPill.addEventListener("click", e => {
   respecPill.setAttribute("aria-expanded", String(menu.hidden));
   menu.hidden = !menu.hidden;
 });
+respecPill.addEventListener("keydown", e => {
+  if (e.key === "Escape") {
+    e.preventDefault();
+    respecPill.click();
```

Comment:
Do we want to reopen the menu by tapping Escape twice? Is it expected elsewhere?

Reply:
Yes , I wanted this happen and also, it is making the code the shorter.
Should I remove this?

Reply:
Reopening already works with Space key, not sure we need this. I think we instead need to focus the first element when users tab Space key, as currently ReSpec just keeps its focus on the pill button.

Reply:
Is closing it by esc key needed?

Reply:
Yes, as otherwise no way to close the menu as we trap the focus within it.